ui/core/src/utils/time-series-data.test.ts

```diff
@@ -0,0 +1,280 @@
+import { TimeScale, TimeSeries, TimeSeriesData } from '../model';
+import { getXValues, getYValues, getCommonTimeScale } from './time-series-data';
+
```

Comment:
We didn't have tests for these utils, so I tried to add a few now that we're exporting these for external consumption.

Reply:
🙏 🙏